src/aiida_quantumespresso/parsers/projwfc.py

```diff
@@ -63,19 +63,16 @@ def parse(self, **kwargs):
 
         orbitals = self._parse_orbitals(header, structure, non_collinear, spinorbit)
         bands, projections = self._parse_bands_and_projections(kpoint_blocks, len(orbitals))
-        energy, dos, pdos_array = self._parse_pdos_files(retrieved_temporary_folder, nspin, spinorbit, logs)
+        energy, dos_node, pdos_array = self._parse_pdos_files(retrieved_temporary_folder, nspin, spinorbit, logs)
+
+        self.out('Dos', dos_node)
```

Comment:
I'd be inclined to change this output name to `dos` (all lowercase), but that is technically backwards-incompatible. However, for the spin-polarised case, by fixing the bug we are also adding an extra array to the `XyData` output, so we are already breaking backwards incompatibility in a sense? Of course changing the output link name would also break backwards incompatibility for the non-spin-polarised case.

Thoughts @t-reents @sphuber @Minotakm @AndresOrtegaGuerrero?

Reply:
How much is the impact on previous calculations , not much right ?

Reply:
I mean the QEapp will definitely break, since it will look for the `Dos` output, not the `dos` one, I assume. ^^

Reply:
we can open an issue to add logic for backwards-compatibility

Reply:
In the end, I think I'll do the inverse: I'll try and preserve backwards compatibility as much as possible, and open an issue to track breaking changes we want to make. I'd also really like to have `PwCalculation.outputs.structure` instead of `PwCalculation.outputs.output_structure` for example. ^^